### ISSUE TITLE
How to configure MarkdownBear

### ISSUE THREAD

**Issue by plaindocs** [coala/documentation#268]
It really isn't clear how to configure bears. I'm trying to configure the MarkdownBear, which is a remark thing, but while remark picks up a `.remarkrc` coala just wipes the files if I try to use it.

**Comment by AbdealiJK** [comment]
True. Are there auto generated  docs for each bear ?

@plaindocs Until then if you do `coala --show-details --show-bears --no-config -b PEP8Bear` this would give you information about the configs and other things about the bear

**Comment by plaindocs** [comment]
Right, that shows me what options are available, but not *actually* how to use them. I dug around a bit in the remark docs, and it wasn't quite gelling.

I had a valid `.remarkrc` file which remark was picking up but coala was just wiping the file.

**Comment by Makman2** [comment]
coala is wiping the file? This shouldn't happen actually. Maybe `remark` does it itself?

About using the settings: You are aware of our .coafile which is our main configuration file? You just specify the setting there:
```
bears = MarkdownBear
markdown_fences = yes
# ...
```

**Comment by plaindocs** [comment]
haha, omg, I was not aware that you'd integrated the settings in the `.coafile`, I thought they were `remark` settings only. I'll make a PR for the MarkdownBear README.

**Comment by plaindocs** [comment]
Looks like the `https://github.com/coala/bear-docs/blob/master/docs/MarkdownBear.rst` is generated? Is there anywhere I can add free text?

**Comment by sils** [comment]
@plaindocs add it at the coala-bears repository directly in the bear, the docs are generated from the documentation comment.

**Comment by sils** [comment]
@plaindocs if you have any feedback for us, including what we just discussed about, you can fill it in at coala.io/usability - we'll definitely look at it.

**Comment by plaindocs** [comment]
Great. We're meeting on Saturday in Berlin anyway I think. :-)

**Comment by sils** [comment]
Ah nice, seeing forward to meet you on 12th then!
